.circleci/config.yml

```diff
@@ -117,22 +116,19 @@ commands:
             ls -l $OUTPUT_DIR
       - deploy-exploded-build:
           study_key: << parameters.study_key >>
-          study_guid: << parameters.study_guid >>
 
   deploy-exploded-build:
     description: Deploy files to GAE
     parameters:
       study_key:
         type: string
-      study_guid:
-        type: string
     steps:
       - set-deployment-environment
       - run:
           name: <<parameters.study_key>> generate config files and copy pepperConfig.js
           command: |
             set -u
-            ./build-study.sh v1 $ENVIRONMENT . <<parameters.study_key>> <<parameters.study_guid>> --config
+            ./build-study.sh v1 $ENVIRONMENT . <<parameters.study_key>> "$STUDY_GUID" --config
             cd ddp-workspace
```

Comment:
They can go away because we are figuring out STUDY_GUID up front and sticking in in an env variable.